migrations/sql/R__stored_procedure_for_bond_etl.sql

```diff
@@ -0,0 +1,358 @@
+CREATE OR REPLACE FUNCTION mms_etl_bond_data() RETURNS void AS $$
+BEGIN
+declare
+
+		tmp_num_records integer;
+        tmp1 integer;
```

Comment:
I think a more descriptive variable name would be helpful here. ETLs are cryptic enough already as is! :P

Reply:
they are reused for print statements, not used in any calculations.

Reply:
Okay, maybe `etl_bond_record_before_count` would help indicate that/make it more readable.

Reply:
these tmp variables get reused, so there isn't a verbose name that is helpful, and making each variable single seems unnecessary.

Reply:
Alright, in that case, I would add a comment by these tmp variables explaining their purpose.